apps/namadillo/src/slices/accounts.ts

```diff
@@ -2,14 +2,14 @@ import { getIntegration } from "@namada/integrations";
 import {
   Account as AccountDetails,
   ChainKey,
-  CosmosTokenType,
   TokenBalances,
   TokenType,
 } from "@namada/types";
 import BigNumber from "bignumber.js";
 import { getSdkInstance } from "hooks";
 import { atom } from "jotai";
 import { atomWithQuery } from "jotai-tanstack-query";
+import { shouldUpdateAmountAtom } from "./etc";
```

Comment:
This is a nitpick, but instead of `Amount` should we use `Balance`? Like `shouldUpdateBalanceAtom`?